config/development.ini

```diff
@@ -135,8 +135,9 @@ solr_url = http://solr:8983/solr/inventory
 # Note: Add ``datastore`` to enable the CKAN DataStore
 #               Add ``resource_proxy`` to enable resource proxying and get around the
 #               same origin policy
-# TODO: re-integrate dcat_usmetadata, usmetadata, datajson, saml2auth
-ckan.plugins = datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars
+# TODO: Figure out how to integrate `saml2auth` since there is no automated testing for
+# that extension as yet.
+ckan.plugins = datagov_inventory datastore xloader stats text_view recline_view googleanalyticsbasic s3filestore envvars dcat_usmetadata usmetadata datajson
```

Comment:
So a weird thing that we should document here; envvars has to be at the end. It has to do with how the plugins are parsed, and making sure that the environment variables are injected properly before other extensions are run.
https://github.com/okfn/ckanext-envvars#requirements